geometry_msgs/Pose target
float64 gripper_pitch
float64 gripper_roll
---
float64 j1
float64 j2
float64 j3
float64 j4
float64 j5
